Docs/Migration-INI.md

```diff
@@ -0,0 +1,854 @@
+# Migrating from older versions - INI configuration
+This guide uses [YR mod base](https://github.com/Starkku/cncnet-client-mod-base) configuration as an example. The majority of changes also applies to to non-YR client configurations.

```

Comment:
Could you write the commit id of the mod base? Since you're refering to a historial state and this repo keeps updating